src/bench/bench.cpp

```diff
@@ -88,7 +88,7 @@ bool benchmark::State::KeepRunning()
     lastCycles = nowCycles;
     ++count;
 
-    if (now - beginTime < maxElapsed) return true; // Keep going
+    if (now - beginTime < maxElapsed || count == 1) return true; // Keep going
```

Comment:
Looks like if `maxElapsed` is > 0 then the logic here is redundant for that code path. `count=0` means `now` and `beginTime` are both 0.